app/models/brew.rb

```diff
@@ -38,6 +48,37 @@ def self.get_last_brewed(limit)
       ]
     }
   end
+  
+  def self.how_to_brew(params)
+    {
+      "text": "Hey #{params["user_name"]}  thanks for asking!",
```

Comment:
Is it necessary to have the person's name show up? I would like it if was anonymous...